public/locales/cs/translation.json

```diff
@@ -196,15 +197,15 @@
 		"Unregister authenticator": "Odhlásit autentikátor",
 		"No authenticator registered": "Žádný autentikátor není registrován",
 		"Are you sure you want to cancel changes?": "Opravdu chcete zrušit změny?",
-		"Key name cannot be empty!": "Název klíče nemůže být prázdný!",
-		"Invalid format, resource cannot start with a number": "Nesprávný formát, zdroj nemůže začínat číslicí",
-		"Invalid format, resource cannot start with a dash": "Nesprávný formát, zdroj nemůže začínat pomlčkou",
-		"Invalid format, resource cannot start with a underscore": "Nesprávný formát, zdroj nemůže začínat podtržítkem",
+		"Authenticator name can't be empty!": "Název autentizátoru nemůže být prázdný!",
```

Comment:
@Aringoaway This is not a `autentizátor` but `autentikátor`. Please be patient and follow already implemented translation rules. Fix it everywhere please